reproducibility_project/src/engines/mcccs/project.py

```diff
@@ -909,6 +919,17 @@ def run_prod(job):
                     job.sp.pressure,
                 )
             )
+            text_file = open("production_information.txt", "w")
+            prod_file = text_file.write(
+                "All prod replicates done. Simulation finished for {} molecule = {}, ensemble = {}, temperature= {} K, pressure = {} kPa.".format(
+                    job,
+                    job.sp.molecule,
+                    job.sp.ensemble,
+                    job.sp.temperature,
+                    job.sp.pressure,
+                )
+            )
+            text_file.close()
```

Comment:
I recommend switching this to a `with open` statement just in case there is a failure while running, the file is guaranteed to be closed.

I would recommend not merging this suggestion though, and instead including it manually using your text editor, since I might have the tabs/spaces incorrect here.
```suggestion
            with open("production_information.txt", "w") as text_file:
                prod_file = text_file.write(
                    "All prod replicates done. Simulation finished for {} molecule = {}, ensemble = {}, temperature= {} K, pressure = {} kPa.".format(
                        job,
                        job.sp.molecule,
                        job.sp.ensemble,
                        job.sp.temperature,
                        job.sp.pressure,
                    )
                )
            text_file.close()
```